source/_static/rtd_theme_overrides.css

```diff
@@ -20,3 +20,22 @@
   margin-left: 0 !important;
 }
 */
+
+/* images/figures with drop shadow */
+.rst-content img.shadowed {
+   box-shadow: 3px 3px 7px hsla(0, 0%, 0%, 0.7);
+}
+
+/* .. figure:: */
+.rst-content div.figure {
+   text-align: center;
+}
+
+.rst-content div.figure p.caption {
+   margin-top: 0.5em;
+
+   font-size: 90%;
+   font-style: italic;
+   font-weight: 100;
+   font-family: "Lato","proxima-nova","Helvetica Neue",Arial,sans-serif;
+}
```

Comment:
No.

The drop shadow is bound to the `shadowed` class which must be specified in the markup, thus making this opt-in. Refer to the usages of `.. image::` and `.. figure::` in `customization/menus.rst`.

The figure-specific styling is just to fix the incredibly ugly default styling of figures.